templates/endpoints_v2/README.md

```diff
@@ -0,0 +1,115 @@
+
+# Step 1 - Deploy the model on Workspace
+
+The serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:
+1. The `models` config in `serve.yaml` contains a list of YAML files for the models you want to deploy. You can any of the models in the `models` directory or define your own model YAML file and run that instead. Follow the CustomModels [guide](CustomModels.md) for bringing your own models.
+2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN environment variable to be set to a Hugging Face Access Token for an account with permissions to download the model.
+
+From the terminal use the Ray Serve CLI to deploy a model:
+
+```shell
+# Deploy the Mistral-7b model. 
+
+serve run serve.yaml
+```
+
+# Step 2 - Query the model
+
+Once deployed you can use the OpenAI SDK to interact with the models, ensuring an easy integration for your applications. Run the following command in a separate terminal to query. 
+
+```shell
+python query.py
+```
+```text
+Output:
+The top rated restaurants in San Francisco include:
+ • Chez Panisse
+ • Momofuku Noodle Bar
+ • Nopa
+ • Saison
+ • Mission Chinese Food
+ • Sushi Nakazawa
+ • The French Laundry
+```
+
+Endpoints uses an OpenAI-compatible API, allowing us to use the OpenAI SDK to access Endpoint backends.
+
+```python
+from openai import OpenAI
+
+client = OpenAI(
+  base_url="http://localhost:8000/v1",
+  api_key="NOT A REAL KEY",
+)
+
+# List all models.
+models = client.models.list()
+print(models)
+
+# Note: not all arguments are currently supported and will be ignored by the backend.
+chat_completion = client.chat.completions.create(
+    model="mistralai/Mistral-7B-Instruct-v0.1",
+    messages=[{"role": "system", "content": "You are a helpful assistant."}, {"role": "user", "content": "What are some of the highest rated restaurants in San Francisco?'."}],
+    temperature=0.01
+)
+
+print(chat_completion)
+
+```
+
+# Step 3 - Deploying a production service - TODO with new CLI
+
+To deploy an application with one model on an Anyscale Service you can run:
+
+```shell
+anyscale service rollout -f service.yaml --name {ENTER_NAME_FOR_SERVICE_HERE}
+```
+
+This is setup to run the Mistral-7B model, but can be easily modified to run any of the other models in this repo.
+
+# Step 4 - Query the service endpoint
+
+In order to query the endpoint, you can modify the `query.py` script, replacing the query url with the Service URL found in the Service UI.
+
+Note: please make sure to include the path "/v1" at the end of the Service url.
+
+Ansycale Services provide highly available fault tolerance for production LLM serving needs.  Learn more about [Anyscale Services](https://docs.anyscale.com/productionize/services/get-started)!
```

Comment:
do we need this line here?

Reply:
Seems unnecessary and confusing for AIOA